packages/auth/package.json

```diff
@@ -13,14 +13,13 @@
   "module": "dist/index.js",
   "types": "dist/index.d.ts",
   "exports": {
-    ".": {
-      "import": "./dist/index.js",
-      "types": "./dist/index.d.ts",
-      "default": "./dist/index.js"
-    }
+    "import": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "default": "./dist/index.js",
+    "legacy": "./dist/legacy/index.js"
```

Comment:
Was there any reason why we set it up with that dot? I always thought it's a bit weird and good if we can get rid off.
Maybe should update common, etc. as well then?

Reply:
> Was there any reason why we set it up with that dot?

I just think it's a confusing copy paste, it's for scoping sub-parts and for sorting CJS/ESM. Some people on the internet does it and I suppose Haffy copy pasted some code from someone that did it. As I understand it's not supposed to be used for the imports, types, and default ones - and we also only create esm bundles. The Nodejs documentation on it: https://nodejs.org/api/packages.html#conditional-exports

> Maybe should update common, etc. as well then?

I'll do that!

Reply:
I guess we'd need to review the old git history before this was public to see? the `player` package does not have that though, so I assume it should work without :-)